plugin/evm/vm.go

```diff
@@ -144,6 +144,7 @@ const (
 	atomicTxGossipProtocol = 0x1
 
 	// gossip constants
+	pushGossipDiscardedElements          = 16_384
```

Comment:
```suggestion
	pushGossipDiscardedElements          = 16 * units.KiB
```

Reply:
I already reverted this change. This is a count, not a size.

Reply:
can we put a comment about why it is specifically this value